proto/user.proto

```diff
@@ -1,8 +1,8 @@
-message GoogleGroupGitHubUserMapping {
-    required string google_user_email = 1;
-    required string git_hub_user_id = 2;
+message User {
+    optional string google_user_email = 1;
+    optional string git_hub_user_id = 2;
 }
 
```

Comment:
The reason for changing to `User` format is: since in the future we will bring in more supported systems, I think it will be easier for users to keep all their user id mappings in one file,  instead of having one file for `GoogleGroup->GitHub`, one for `GoogleGroup->GitLab` etc...

Reply:
nit: I like the name `UserMapping` better than `User`

Reply:
Renamed to `UserMapping`